requirements.txt

```diff
@@ -1,2 +1,3 @@
 sphinx
 sphinx_rtd_theme
+sphinxcontrib-mermaid
```

Comment:
oh cool, i guess that this add support for the mermaid block you added above! great, if it's tested then you can ignore my comments there